OAuthSDK/YMAPIClient.h

```diff
@@ -23,20 +23,27 @@
 #import <Foundation/Foundation.h>
 #import "AFNetworking.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 extern NSString * const YMBaseURL;
 
+typedef void(^successCallback)(id __nullable responseObject);
+typedef void(^failureStatusCodeCallback)(NSInteger statusCode, NSError *error);
+typedef void(^failureCallback)(NSError *error);
+
 /**
  Represents an object that contains a queue of HTTP operations.
  At the moment, this is essentially a lightweight wrapper around AFHTTPSessionManager.
  */
 @interface YMAPIClient : NSObject
-@property (nonatomic, copy) NSString *authToken;
+
+@property (nonatomic, copy, nullable) NSString *authToken;
 
 /**
  Default initializer.
  @param authToken The OAuth token.
  */
-- (id)initWithAuthToken:(NSString *)authToken;
+- (instancetype)initWithAuthToken:(nullable NSString *)authToken;
```

Comment:
I think we should be consistent and use the order Apple's using which I think is `type nullability_annotation`.